exercises/perfect-numbers/tests/perfect-numbers.rs

```diff
@@ -0,0 +1,35 @@
+extern crate perfect_numbers;
+
+use perfect_numbers::{Classification, classify};
+
+#[test]
+fn basic() {
+    assert_eq!(classify(0).unwrap_err(), "Number must be positive");
```

Comment:
by the request of #338, I think this should be `assert_eq!(classify(0), Err("Number must be positive"))` since that will display a better error message to the student if the function under test incorrectly returns an `Ok`. Can you confirm whether this true, and if it is true do change it?

Reply:
Comparison changed to Ok(value)